packages/common/mui/src/lib/product-dashboard/product-dashboard.tsx

```diff
@@ -21,6 +21,7 @@ export interface DashboardDataProps {
   bottomLeftValue?: string | ReactNode;
   bottomRightValue?: string | ReactNode;
   hasPosition?: boolean;
+  isPointsVault?: boolean;
```

Comment:
We will likely be extending the vault types in the future, can we make this an enum instead?
"Leveraged Yield Farming"
"Leveraged Points Farming"

etc.